src/backend/src/server.js

```diff
@@ -0,0 +1,47 @@
+import { MapeoManager } from '@mapeo/core'
+import { createMapeoServer } from '@mapeo/ipc'
+import debug from 'debug'
+
+import MessagePortLike from './message-port-like.js'
+import { STATUSES, ServerStatus } from './status.js'
+
+const log = debug('mapeo:server')
+
+export class Server {
+  #channel
+  #mapeoManager
+  #status
+  /** @type {{ close: () => void } | null} */
+  #apiServer = null
+
+  /**
+   * @param {MapeoManager} mapeoManager
+   */
+  constructor(mapeoManager) {
+    this.#channel = new MessagePortLike()
+    this.#mapeoManager = mapeoManager
+    this.#status = new ServerStatus()
```

Comment:
figured it makes sense to couple the status to this server instance, but not entirely sure. in mapeo mobile, they're kept separate but the coordination of their calls is similar to what's done in this class, hence my decision to colocate. open to input though